app/models/json_schemas/stop.json

```diff
@@ -13,6 +13,9 @@
     "timezone": { "type": "string" },
     "tags": { "type": "object" },
     "name": { "type": "string" },
+    "parentStopOnestopId": {
+      "type": ["string", "null"]
```

Comment:
Is it possible to specify format as the stop Onestop ID regex validator? Or is that not compatible with the combo of string-or-null?